exercise/api/serializers.py

```diff
@@ -40,6 +40,7 @@ class Meta(AplusModelSerializer.Meta):
         model = Submission
         fields = (
             'submission_time',
+            'grade',
```

Comment:
What does this change do and was it required for changing the course points endpoint?

Reply:
Probably not necessary anymore, I will remove it

Reply:
I think this would be fine too. Then the "brief" would include the same information in all locations.

Reply:
Including useful info is OK. I wanted to check if this was really needed for the actual issue, but let's keep it now if there are no unexpected consequences from this.